lucene/misc/native/build.gradle

```diff
@@ -51,11 +47,7 @@ tasks.withType(CppCompile).configureEach {
   // is present.
   systemIncludes.from file("${javaHome}/include")
 
-  for (def path : [
```

Comment:
I'd leave it in as it was (multiplatform includes, conditionally) - doesn't hurt even if we only compile on a single platform and may save somebody some head-scratching in the future.

Reply:
> Sorry for dropping off the radar a bit. It looks good to me. It is a pity we can't drop native builds on Windows too!

No problem, and thanks again for your review and comment on this PR! I would feel sad to see all of native build code gone at once actually. So this helps soften the blow :D

Reply:
"Soften the blow", you say? :) Even the cover of the album this song is from kind of reflects the nature of open source, doesn't it? 
https://www.youtube.com/watch?v=G1pfrCOQWHY

Reply:
Hahahahahaha! YES DEFINITELY! Good one Dawid!